AssetInformationApi.Tests/SnsEventVerifier.cs

```diff
@@ -0,0 +1,128 @@
+using Amazon.SimpleNotificationService;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace AssetInformationApi.Tests
```

Comment:
You can use the SnsEventVerifier in the Hackney nuget package - https://github.com/LBHackney-IT/lbh-core/blob/release/Hackney.Core/Hackney.Core.Testing/Hackney.Core.Testing.Sns/SnsEventVerifier.cs

Reply:
Thanks. Changed